locality-checker/pkg/kubernetes/clientset.go

```diff
@@ -0,0 +1,19 @@
+package kubernetes
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func BuildClientset() (kubernetes.Interface, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "error building kubernetes cluster config")
+	}
```

Comment:
nit: For testing, it would be useful to try to load kubectl config at this point.